src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Offender.kt

```diff
@@ -120,6 +99,11 @@ data class Offender(
   val middleNames: String
     get() = StringUtils.trimToNull(StringUtils.trimToEmpty(middleName) + " " + StringUtils.trimToEmpty(middleName2))
 
+  fun getAllBookings(): MutableList<OffenderBooking>? = rootOffender?.allBookings
+
+  fun latestBooking(): OffenderBooking =
+    getAllBookings()?.firstOrNull { it.active } ?: throw IllegalStateException("Offender has no active bookings")
```

Comment:
I think the isActive check isn't appropriate - I think latestBooking should do what it says on the tin.
do we need `activeBooking`

Also latestBooking should simply be find {it.bookingSequence == 1} no need for any ordering